testing/api/test_api.py

```diff
@@ -265,6 +482,268 @@ def test_status_in_progress(testing_devices, testrun):  # pylint: disable=W0613
       600,
   )
 
+# Currently not working due to blocking during monitoring period
+@pytest.mark.skip()
```

Comment:
We don't need to try starting testrun again in the in progress state, we can test this when testrun is in the waiting for device state.